CHANGELOG_PENDING.md

```diff
@@ -1,8 +1,6 @@
 ### Improvements
 
-- Classloader: now supports dynamic exclusions of packages from system property.
+- Add support for attaching debuggers
 
 ### Bug Fixes
 
-- Fix HTML escaping after `{@literal ...}` blocks in Javadocs
-- Fix issue with direct dependencies causing IndexOutOfBoundsException
```

Comment:
Doesn't look like this line is in CHANGELOG.md

Reply:
Good catch. Fixed up CHANGELOG_PENDING.md and CHANGELOG.md.

Reply:
We should probably use changie to stop cases like this....

Reply:
> We should probably use changie to stop cases like this....

Yes